collector/conntrack_linux.go

```diff
@@ -60,5 +103,51 @@ func (c *conntrackCollector) Update(ch chan<- prometheus.Metric) error {
 	ch <- prometheus.MustNewConstMetric(
 		c.limit, prometheus.GaugeValue, float64(value))
 
+	data, err := ioutil.ReadFile(procFilePath("net/stat/nf_conntrack"))
+	if err != nil {
+		return nil
+	}
+	connStat := parseNfConntrackStat(data)
+	ch <- prometheus.MustNewConstMetric(
+		c.searched, prometheus.GaugeValue, float64(connStat.Searched))
+	ch <- prometheus.MustNewConstMetric(
+		c.insert, prometheus.GaugeValue, float64(connStat.Insert))
+	ch <- prometheus.MustNewConstMetric(
+		c.insertFailed, prometheus.GaugeValue, float64(connStat.InsertFailed))
+	ch <- prometheus.MustNewConstMetric(
+		c.delete, prometheus.GaugeValue, float64(connStat.Delete))
+	ch <- prometheus.MustNewConstMetric(
+		c.drop, prometheus.GaugeValue, float64(connStat.Drop))
 	return nil
 }
+
+func parseNfConntrackStat(data []byte) *ConntrackStatistics {
+	connstat := ConntrackStatistics{}
+
+	r := bytes.NewReader(data)
+	scanner := bufio.NewScanner(r)
+
+	for scanner.Scan() {
+		fields := strings.Fields(scanner.Text())
+		parseFields(&connstat, fields)
+	}
+
+	return &connstat
+}
+
+func parseFields(cs *ConntrackStatistics, fields []string) {
+	searched, _ := strconv.ParseUint(fields[1], 16, 64)
```

Comment:
Even if unlikely, we should still handle these errors.

Reply:
Done!